src/api/uploads/__tests__/PlainUpload-test.js

```diff
@@ -228,8 +228,11 @@ describe('api/uploads/PlainUpload', () => {
             const folderId = '123';
             const file = {};
             const successCallback = () => {};
+
```

Comment:
This is annoying

Reply:
agreed we should skip one liners. Working on it